### ISSUE TITLE
[bug] twin-drift truncates scoped script names (test:unit shown as test)

### ISSUE THREAD

**Issue by cemililik** [alifurkangokce/driftlint#15]
### What happened

`twin-drift` builds its command keys as `kind:name`, then renders them with `split(":")[1]` — which takes only the **second** segment. Scoped script names lose everything after the first colon, so the message names a script that does not exist.

```ts
// src/checks/twinDrift.ts:73
const cmdSet = (t: TwinInput) => new Set(t.commands.map((c) => `${c.kind}:${c.name}`));

// src/checks/twinDrift.ts:81
`${list.slice(0, MAX_EXAMPLES).map((c) => c.split(":")[1]).join("`, `")}`
```

`npm-script:test:unit` renders as `test`.

The detection itself is correct — the set comparison on lines 76-77 uses the full string, so there is no false positive or negative here. Only the message is wrong, and it points the reader at a script that is not in their `package.json`. `dead-command` stays quiet (the real script *does* exist), so nothing contradicts the wrong name.

### How to reproduce

```
repo/
├── CLAUDE.md      # "Run `npm run test:unit` before pushing." + "Also `npm run lint:fix`."
├── AGENTS.md      # "Run `npm run check:types` before pushing." + "Also `npm run lint:fix`."
└── package.json   # scripts: test:unit, lint:fix, check:types
```

```console
$ driftlint .
CLAUDE.md and AGENTS.md have drifted apart — command claims differ: `test` only in
CLAUDE.md; `check` only in AGENTS.md; the files are 80% identical but 2 lines differ.
```

Expected `test:unit` and `check:types`. There is no `test` or `check` script anywhere in the repo.

`grep -rn 'split(":")' src/` returns exactly one line, so the fix is a single point:

```ts
c.slice(c.indexOf(":") + 1)
```

Colon-free names are unaffected (`make-target:build` -> `build` either way), and the existing `test/fixtures/twins/diverged/` fixture only uses colon-free names, so `test/v10.test.js` passes unchanged — which is also why this was never caught.

### driftlint version and OS

0.14.1 (main @ 81ffeae), macOS 26.5.2 / node 22.23.1
